src/server/handlers/utils/util_filesys.hpp

```diff
@@ -56,7 +56,9 @@ void write_entry_dir(int tid, const std::string &file_path, const std::string &d
     START_LOG(tid, "call(file_path=%s, dir=%s, type=%d)", file_path.c_str(), dir.c_str(), type);
 
     std::hash<std::string> hash;
-    struct linux_dirent64 ld;
+    struct linux_dirent64 ld {
+        0
```

Comment:
Why do you pass `0` instead of simply using the empty constructor `linux_dirent64 ld{}`?

Reply:
No real reason, other than to set all values to zero. Will change with default contructor